.drone.yml

```diff
@@ -58,6 +58,24 @@ steps:
     event:
     - tag
 
+- name: build_tag_push_quay
```

Comment:
Do we still need the previous build_tag_push_ecr step if we are using the new quay repo in the values files?

Reply:
Will remove it. I left it to test only. Thanks